### PR TITLE
fk: Add install instructions

### PR DESCRIPTION
**Pull request by seisman**
**Description of proposed changes**

Related to #39.

**PRs for new software/tutorials**

- [x] Software metadata is complete
- [x] The "last updated date" in the metadata page is up-to-date
- [x] All links work
- [x] You're listed as the author(s) in the metadata page
- [ ] Anyone who have made significant contributions (e.g., review) to
      the tutorials are listed in the meteda page

**PRs for bugs/improvements**

- [ ] You're listed as the proofreader in the metadata page

### CODE REVIEW

File: source/fk/install.rst [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        使得 Fortran 一行可以超过72字符
+    -   :file:`Makefile` 中第 6—7 行被注释了，所以 ``fk`` 无法使用 SAC 提供的
+        滤波功能。若 SAC 已安装，则可以注释将第 6—7 行行首的注释符号 ``#`` 去掉
+    -   :file:`Makefile` 中未指定如何编译生成 ``fk2mt`` 文件。需要将 ``fk2mt``
+        加到变量 ``TARGETS`` 中，并向 :file:`Makefile` 尾部加入如下语句：
+
+        .. code-block:: makefile
+
+            fk2mt: fk2mt.o sacio.o radiats.o
+                $(LINK.f) -o $@ $^ -lm
```
Comment (core-man) [inline source/fk/install.rst:33]: Do we need to add a blank line here?

File: source/fk/install.rst [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        $ mv fk ~/src/
+
+3.  修改 fk 源码
+
+    fk3.3（下载于 2021-01-13，软件包中文件的最新日期为 2019-06-18）的原始代码
+    存在一些问题，因而需要做一些小修改方可使用。主要修改如下：
+
+    -   :file:`Makefile` 中需要添加 ``FC=gfortran`` 指定使用 gfortran 编译器
+    -   :file:`Makefile` 中需要给 ``FFLAGS`` 加上中 ``-ffixed-line-length-none``
+        使得 Fortran 一行可以超过72字符
```
Comment (core-man) [inline source/fk/install.rst:24]:
```suggestion
        使得 Fortran 一行可以超过 72 字符
```

File: source/fk/install.rst [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    -   :file:`syn.c` 第 142 行需改写为 ``mt[0][1]=mt[0][2]=mt[1][2] = 0.;``
+    -   :file:`sac.h` 中 319 行 ``sac_head_inex`` 应改为 ``sac_head_index``
+
+    .. note::
+
+        为了便于用户使用，我们对 fk 源码做了以上修改，并提供了 patch 文件，供用户使用。
+        下载 patch 文件 :download:`fk3.3-v20190618.patch`，将其放在 fk 源码目录下，
+        然后执行如下命令即可修改源码::
+
+            $ patch < fk3.3-v20190618.patch
```
Comment (core-man) [inline source/fk/install.rst:43]: Amazing! How to make such a patch file?

File: source/fk/install.rst (hunk 1 above)
Comment (seisman) [reply source/fk/install.rst:33]: Where? Between line 32 and 33?

Comment (core-man) [reply source/fk/install.rst:33]: 33 and 34

File: source/fk/install.rst (hunk 3 above)
Comment (seisman) [reply source/fk/install.rst:43]: 1. Download the fk3.3 package
2. In the fk directory, run `git init` and add all files to git. This is the initial buggy version
3. Make some changes to the codes, and make sure it works
4. Revert changes of some unrelated files (e.g., `fk`, `syn` are rebuilt)
5. Run `git diff --patch > fk3.3.patch` to create the patch

File: source/fk/install.rst (hunk 1 above)
Comment (seisman) [reply source/fk/install.rst:33]: Yes, it's better. Now fixed.

File: source/fk/install.rst (hunk 3 above)
Comment (core-man) [reply source/fk/install.rst:44]: Nuibitliy~

File: source/fk/install.rst [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+    .. note::
+
+        我们已经将以上 BUG 报告给原作者，在新版本中这些问题可能会得到修复。
+
+4.  进入源码目录并编译::
+
+        $ cd fk/
+        $ make
```
Comment (core-man) [inline source/fk/install.rst:53]: This will create some *.o files.  If we run `make clean`, they will be deleted but the bessel.f will be deleted.

Comment (seisman) [reply source/fk/install.rst:53]: `bessel.f` is a temporary file generated from `bessel.FF`, so it's safe to remove it.

Comment (core-man) [reply source/fk/install.rst:53]: Shall we add `make clean`?

Comment (seisman) [reply source/fk/install.rst:53]: Done.

**Review by core-man — APPROVED** [review]

**Pull request merged**